### PR TITLE
feat: Allow endpoint url to be passed through to aws clients

### PR DESCRIPTION
**Pull request by mikeforkin**
I am migrating from aws sso to teleport and connecting to ecs containers through a teleport proxy requires this configuration to be specified.

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -94,6 +94,9 @@ func init() {
 	rootCmd.PersistentFlags().BoolP("forward", "f", false, "Port Forward")
 	rootCmd.PersistentFlags().StringP("local-port", "l", "", "Local port for use with port forwarding")
 	rootCmd.PersistentFlags().BoolP("quiet", "q", false, "Do not print cluster and container information")
+	rootCmd.PersistentFlags().StringP("aws_endpoint_url", "e", "", "AWS Endpoint Url")
+
+	viper.AutomaticEnv()
```
[the remark this answers is not in the document]
Comment (tedsmitt) [reply cmd/main.go:101]: Happy to have this as a CLI opt

[the remark this answers is not in the document]
Comment (mikeforkin) [reply cmd/main.go:101]: Sounds good, I'll update

[the remark this answers is not in the document]
Comment (mikeforkin) [reply cmd/main.go]: Updated, will resolve this comment, anything new can go in a separate one.

**Review by tedsmitt — APPROVED** [review]
Have tested locally using this branch, looks to work as expected. Approving.

**Comment by tedsmitt** [comment]
@mikeforkin if I was to remove the `enable-env` flag and then allow optional parameters to be passed via prefixed env vars e.g. `ECSGO_AWS_ENDPOINT_URL`, `ECSGO_CLUSTER` etc. Would that also work for your use case?

**Pull request merged**